install/kubernetes/helm/istio/charts/certmanager/templates/crds.yaml

```diff
@@ -0,0 +1,56 @@
+{{- if .Values.createCustomResource -}}
```

Comment:
THanks for merging the CRDs.  Because of the way Istio installs (we do not want a hard dependency on tiller) we use `helm template` to create our templates.  The concept of `--set certmanager.createCustomResource=true` requires two runs of the chart - and two unique templates.  The technique we use to solve this problem in Istio is a post-install job.  That looks a lot like this:
https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/mixer/templates/create-custom-resources-job.yaml

Can you explain the workflow of using the flag on line 1?

Reply:
You're right. The flag setting is not needed.